cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -151,13 +164,24 @@ func (i *deployCommand) InitHook() {
 	i.command.Flags().StringVar(&i.flags.imageVersion, "image-version", "", "Image version for standard Kogito build images. Ignored if a custom image is set for image-s2i or image-runtime.")
 	i.command.Flags().StringVar(&i.flags.mavenMirrorURL, "maven-mirror-url", "", "Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed, e.g: https://my.internal.nexus/content/group/public")
 	i.command.Flags().BoolVar(&i.flags.enableIstio, "enable-istio", false, "Enable Istio integration by annotating the Kogito service pods with the right value for Istio controller to inject sidecars on it. Defaults to false")
+	i.command.Flags().StringVarP(&i.flags.fromFile, "from-file", "f", "", "If set, a build from a single file, no pom.xml needed, will start, you can provide a entire directory, a single file from filesystem or url, the supported files are dmn, drl, bpmn or bpmn2. Porperties files is also allowed.")
```

Comment:
I would change the description to "If set, a build from local sources will start (no pom.xml needed). You can provide a entire directory, a single file from filesystem or url, the supported files are dmn, drl, bpmn or bpmn2. Properties files is also allowed."

Also, note the typo in the properties word.

Reply:
cool, thanks.